### DISCUSSION TITLE
Disable animations / transition effects of components

### DISCUSSION THREAD

**Discussion by tomsoja978** [primefaces/community#902] [PrimeReact] [answered]
How can I disable the animations and/or transition effects of components? Like ProgressBar, LineChart, for instance.

https://primereact.org/chart/

**Comment by melloware** [comment, marked as answer]
See this open ticket: https://github.com/primefaces/primereact/issues/5605

For ChartJS just turn `animation:false` like the docs say: https://www.chartjs.org/docs/latest/configuration/animations.html#disabling-animation

**Reply by tomsoja978** [reply]
Thanks. Worked like a charm! Can you please explain me how I can achieve the same with a ProgressBar? https://primereact.org/progressbar/

**Reply by melloware** [reply]
ProgressBar you need the ticket https://github.com/primefaces/primereact/issues/5605 implemented to turn it off.

**Reply by tomsoja978** [reply]
I would like to disable the animation regardless of the prefers-reduced-motion-settings. can you please explain me how i have to accomplish that in detail (sorry, I am new to css and primereact )

**Reply by melloware** [reply]
Unfortunately that ticket needs to be solved on disabling animations.  Right now there is no global way to disable animations. I sent you that ticket so you could +1 and subscribe.

**Reply by tomsoja978** [reply]
I don't understand what you mean. Really sorry. Does the user have to change some windows os settings!? I can not ask them to do that :(

if you meant something else, I need an easy, detailed, step-by-step instructions since i am very new to this world. Thank you very much for understanding.

**Reply by melloware** [reply]
It means that ticket is still open so it's not solved yet.
